packages/elements/src/components/ino-card/ino-card.tsx

```diff
@@ -40,7 +40,7 @@ export class Card implements ComponentInterface {
     return (
       <Host>
         <div class={classList}>
-          <div>
+          <div style={{"display": "contents"}}>
```

Comment:
We should be using a class instead of this inline-style

Reply:
✅